core/backend/src/workspace/Workspace.ts

```diff
@@ -102,22 +88,27 @@ export namespace WorkspaceContainer {
    * Parse the name stored in a WorkspaceContainer into the dbName and version number. A single WorkspaceContainer may hold
    * many versions of the same WorkspaceDb. The name of the Db in the WorkspaceContainer is in the format "name:version". This
    * function splits them into separate strings.
+   * @internal
    */
   export function parseDbFileName(dbFileName: WorkspaceDb.DbFullName): { dbName: WorkspaceDb.DbName, version: WorkspaceDb.Version } {
     const parts = dbFileName.split(":");
     return { dbName: parts[0], version: parts[1] };
   }
 
-  /** Create a dbName for a WorkspaceDb from its base name and version. This will be in the format "name:version" */
+  /** Create a dbName for a WorkspaceDb from its base name and version. This will be in the format "name:version"
+   * @internal
+   */
   export function makeDbFileName(dbName: WorkspaceDb.DbName, version?: WorkspaceDb.Version): WorkspaceDb.DbName {
     return `${dbName}:${WorkspaceContainer.validateVersion(version)}`;
   }
 }
 
 /** @beta */
 export namespace WorkspaceDb {
-  /** The name of a WorkspaceDb in a "workspace/databases" setting. */
-  export type Name = string;
+  /** complete a WorkspaceDb name, replacing undefined with the default value "workspace-db"
+   * @internal
+   */
+  export const dbNameWithDefault = (dbName?: WorkspaceDb.DbName) => dbName ?? "workspace-db";
```

Comment:
Most likely WorkspaceContainers will hold one and only one WorkspaceDb (i.e. they are 1:1). The previous api had "workspace db name" as a required field in many places. To make the most common scenario easier, there is now a "default name" (workspace-db) that is used if no name is provided.